src/cryptonote_core/blockchain.cpp

```diff
@@ -433,9 +433,9 @@ bool Blockchain::init(BlockchainDB* db, const network_type nettype, bool offline
   uint64_t top_block_timestamp = m_db->get_top_block_timestamp();
   uint64_t timestamp_diff = time(NULL) - top_block_timestamp;
 
-  // genesis block has no timestamp, could probably change it to have timestamp of 1341378000...
+  // genesis block has no timestamp, could probably change it to have timestamp of 1402185600...
   if(!top_block_timestamp)
-    timestamp_diff = time(NULL) - 1341378000;
+    timestamp_diff = time(NULL) - 1402185600;
```

Comment:
What does this timestamp reference? Isn't the original genesis around1397818150 ?

Reply:
date  --date=@1402185600
Sun Jun  8 00:00:00 GMT 2014

From account.cpp:

    timestamp.tm_year = 2014 - 1900;  // year 2014
    timestamp.tm_mon = 6 - 1;  // month june
    timestamp.tm_mday = 8;  // 8th of june

Reply:
Indeed, block 1 shows 18th of april. I assumed the code was correct.

Reply:
Fixed.